### PR TITLE
Release 0.14

### PR DESCRIPTION
**Pull request by maroshii**
Note: do not merge until #130 gets merged onto `release-0.14`

### CODE REVIEW

**Comment by pchico83** [comment]
@rberrelleza could you review the release notes? I think that part didn't have a previous PR:
https://github.com/okteto/docs/pull/134/files#diff-fdc6ee7e265beb2f6428769ee199f648a54ed18b263b1d44c154d9bbe004969f

File: src/content/cloud/deploy-from-helm.mdx [hunk 1]
```diff
@@ -10,6 +10,9 @@ import Image from '@theme/Image';
 You can launch additional services for your development environments on Okteto Cloud using Helm repositories.
 Okteto automates all the processes needed to deploy, configure, upgrade, and destroy your Helm releases, so you can focus on building amazing applications.
 
+You can also list your own charts under the [Helm Catalog](/docs/enterprise/administration/configuration/#applications) tab of the **Launch Dev Environment** menu.
```
Comment (rberrelleza) [inline src/content/cloud/deploy-from-helm.mdx:13]: Can we add a subheader for this? (e.g. "Add your own charts") and the TierList badge for 'Self-hosted'?

File: src/content/enterprise/administration/configuration.mdx [hunk 2]
```diff
@@ -460,7 +460,7 @@ In this case, pods deployed in namespaces created by Okteto will have a preferre
 
 ### applications
 
-- `repository`: The default [application repository](/docs/tutorials/repos/) for every Okteto Enterprise user. It defaults to `https://apps.okteto.com` when not specified.
+- `repository`: The [Helm repository](/docs/cloud/deploy-from-helm/) available for every developer. By default, deploying from a Helm repository isn't available.
```
Comment (rberrelleza) [inline src/content/enterprise/administration/configuration.mdx:487]: how do we enable/disable this feature? Can we add that on the doc?

```suggestion
- `repository`: The [Helm repository](/docs/cloud/deploy-from-helm/) available for every developer. This is disabled by default. To enable, set the `XXXX` property to `YYYY`
```

File: src/content/enterprise/install/releases.mdx [hunk 3]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.14.0
+TBD
+
+### Features
+- Add structured and staged logs for pipelines. Groups logs by stage in the okteto UI.
+- Support custom envvars for main okteto components
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:13]:
```suggestion
- Support adding custom environment variables for main okteto components
```

File: src/content/enterprise/install/releases.mdx [hunk 4]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Support defining custom annotation, labels and envvars in pipeline installer jobs
+- Display CLI version for pipeline installer jobs in the Okteto UI admin panel
+- Add ability to download the Okteto cluster context from the UI
+
+### Improvements
+- Upgrade Okteto CLI to [2.4.1](https://github.com/okteto/okteto/releases/tag/2.4.1)
+- Add better defaults for timestamps in event logs
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:20]: can we be more specific on this? What is the bug that we fixed?

File: src/content/enterprise/install/releases.mdx [hunk 5]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Remove deprecated Google Sign-in Javascript Platform Library in favor or standard OAuth. More info [here](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html)
+- Remove the [helm catalog](https://www.okteto.com/docs/enterprise/administration/configuration/#applications) on self-hosted installations by default. This makes installing CRDs not mandatory during cluster creation.
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:22]: Are we updating the installation guides to reflect this?

```suggestion
- Disable the [helm catalog](https://www.okteto.com/docs/enterprise/administration/configuration/#applications) on self-hosted installations by default. This makes installing CRDs not mandatory during cluster creation.
```

File: src/content/enterprise/install/releases.mdx [hunk 6]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+
+
+### Bugfixes
+- Improve backend api upgrades
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:26]: Can we be more specific on this?

File: src/content/enterprise/install/releases.mdx [hunk 7]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Do not attempt to redeploy snapshots in the webhook for update operations
+- Account for installer jobs token in registry auth
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:28]: What does this mean?

File: src/content/enterprise/install/releases.mdx [hunk 8]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Block deploy button in UI while request is in-flight. This was causing duoble deploys sometimes.
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:29]:
```suggestion
- Block the deploy button in the UI while requests are in-flight. This was causing double deploys in some scenarios.
```

File: src/content/enterprise/install/releases.mdx [hunk 9]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Fix memory leak caused by not freeing up resoures sometime after closing the connection when streaming logs.
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:30]: Can we add on which resource this is? Is it in the webapp?
```suggestion
- Fixed memory leak caused by not freeing up resoures sometime after closing the connection when streaming the logs.
```

File: src/content/enterprise/install/releases.mdx [hunk 10]
```diff
@@ -10,13 +10,13 @@ TBD
 
 ### Features
 - Add structured and staged logs for pipelines. Groups logs by stage in the okteto UI.
-- Support custom envvars for main okteto components
-- Support defining custom annotation, labels and envvars in pipeline installer jobs
-- Display CLI version for pipeline installer jobs in the Okteto UI admin panel
-- Add ability to download the Okteto cluster context from the UI
+- Support adding custom environment variables for main okteto components
```
Comment (RinkiyaKeDad) [inline src/content/enterprise/install/releases.mdx:13]:
```suggestion
- Support adding custom environment variables for main Okteto components
```

File: src/content/enterprise/install/releases.mdx [hunk 11]
```diff
@@ -26,7 +26,7 @@ TBD
 - Improve backend api upgrades
 - Do not attempt to redeploy snapshots in the webhook for update operations
 - Account for installer jobs token in registry auth
-- Block deploy button in UI while request is in-flight. This was causing duoble deploys sometimes.
+- Block the deploy button in the UI while requests are in-flight. This was causing double deploys in some scenarios.
```
Comment (RinkiyaKeDad) [inline src/content/enterprise/install/releases.mdx:29]:
```suggestion
- Blocked the deploy button in the UI while requests are in-flight. This was causing double deploys in some scenarios.
```

File: src/content/enterprise/install/releases.mdx (hunk 7 above)
Comment (maroshii) [reply src/content/enterprise/install/releases.mdx:28]: This is https://github.com/okteto/app/pull/4077

@pchico83 how can we put this?

Comment (pchico83) [reply src/content/enterprise/install/releases.mdx:27]: @rberrelleza the token we were using in installer jobs wasn't supported in our registry auth layer. this was breaking the use case to use our helm registry as a hemp chart repository

File: src/content/enterprise/install/releases.mdx (hunk 6 above)
Comment (maroshii) [reply src/content/enterprise/install/releases.mdx]: Thanks @jpf-okteto! Renamed it and moved it to the improvements section

File: src/content/enterprise/administration/configuration.mdx [hunk 12]
```diff
@@ -460,7 +484,7 @@ In this case, pods deployed in namespaces created by Okteto will have a preferre
 
 ### applications
 
-- `repository`: The default [application repository](/docs/tutorials/repos/) for every Okteto user. It defaults to `https://apps.okteto.com` when not specified.
+- `repository`: The [Helm repository](/docs/cloud/deploy-from-helm/) available for every developer. This is disabled by default. To enable, set the `XXXX` property to `YYYY`
```
Comment (pchico83) [inline src/content/enterprise/administration/configuration.mdx:487]:
```suggestion
- `repository`: The [Helm repository](/docs/cloud/deploy-from-helm/) available for every developer. This is disabled by default. To enable, set the `applications.repository` property to your Helm registry URL
```

File: src/content/enterprise/install/releases.mdx [hunk 13]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.14.0
+TBD
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:9]:
```suggestion
July 14, 2022
```

File: src/content/enterprise/install/releases.mdx [hunk 14]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+
+### Features
+- Add structured and staged logs for pipelines. Groups logs by stage in the okteto UI.
+- Support adding custom environment variables for main Okteto components
+- Support adding custom annotations, labels, and environment variables to the pipeline installer jobs
+- Display the CLI version used by the pipeline installer jobs in the Okteto UI admin panel
+- You can now download the Okteto cluster context from the UI
+
+### Improvements
+- Upgraded Okteto CLI to [2.4.1](https://github.com/okteto/okteto/releases/tag/2.4.1)
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:19]: https://demo.okteto.dev/ says we have 2.4.2. Can you double check the version?

File: src/content/enterprise/install/releases.mdx [hunk 15]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Remove deprecated Google Sign-in Javascript Platform Library in favor or standard OAuth. More info [here](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html)
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:20]:
```suggestion
- Remove deprecated Google Sign-in Javascript Platform Library in favor or standard OAuth. More info [here](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html). No action is required. 
```

File: src/content/enterprise/install/releases.mdx (hunk 7 above)
Comment (rberrelleza) [reply src/content/enterprise/install/releases.mdx:27]: can we phrase it that way then? The way is described today doesn't tell me anything as a user or admin of okteto.

File: src/content/enterprise/install/releases.mdx [hunk 16]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Disable the [helm catalog](https://www.okteto.com/docs/enterprise/administration/configuration/#applications) on self-hosted installations by default. This makes installing CRDs not mandatory during cluster creation.
+- Improve K8S rolling update process for Okteto API deployment
+
+
+### Bugfixes
+- Do not attempt to redeploy snapshots in the webhook for update operations
+- Account for installer jobs token in registry auth
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:27]: Is this accurate?
```suggestion
- Pipelines are now able to push and pull charts to the internal repository
```

File: src/content/enterprise/install/releases.mdx [hunk 17]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Block the deploy button in the UI while requests are in-flight. This was causing double deploys in some scenarios.
+- Fixed memory leak caused by not freeing up resoures sometime after closing the connection when streaming the logs.
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:29]:
```suggestion
- Fixed memory leak caused by not freeing up some resources after closing the connection when streaming the logs.
```

File: src/content/enterprise/install/releases.mdx [hunk 18]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Upgraded Okteto CLI to [2.4.1](https://github.com/okteto/okteto/releases/tag/2.4.1)
+- Remove deprecated Google Sign-in Javascript Platform Library in favor or standard OAuth. More info [here](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html)
+- Disable the [helm catalog](https://www.okteto.com/docs/enterprise/administration/configuration/#applications) on self-hosted installations by default. This makes installing CRDs not mandatory during cluster creation.
+- Improve K8S rolling update process for Okteto API deployment
+
+
+### Bugfixes
+- Do not attempt to redeploy snapshots in the webhook for update operations
+- Account for installer jobs token in registry auth
+- Block the deploy button in the UI while requests are in-flight. This was causing double deploys in some scenarios.
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:28]:
```suggestion
- The deploy button is now disabled in the UI while requests are in-flight. This was causing double deploys in some scenarios.
```

File: src/content/enterprise/install/releases.mdx [hunk 19]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+
+### Improvements
+- Upgraded Okteto CLI to [2.4.1](https://github.com/okteto/okteto/releases/tag/2.4.1)
+- Remove deprecated Google Sign-in Javascript Platform Library in favor or standard OAuth. More info [here](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html)
+- Disable the [helm catalog](https://www.okteto.com/docs/enterprise/administration/configuration/#applications) on self-hosted installations by default. This makes installing CRDs not mandatory during cluster creation.
+- Improve K8S rolling update process for Okteto API deployment
+
+
+### Bugfixes
+- Do not attempt to redeploy snapshots in the webhook for update operations
```
Comment (rberrelleza) [inline src/content/enterprise/install/releases.mdx:26]:
```suggestion
- The webhook will not redeploy snapshots as part of an update operation.
```

File: src/content/enterprise/install/releases.mdx (hunk 14 above)
Comment (maroshii) [inline src/content/enterprise/install/releases.mdx:19]:
```suggestion
- Upgraded Okteto CLI to [2.4.2](https://github.com/okteto/okteto/releases/tag/2.4.2)
```

**Review by rberrelleza — APPROVED** [review]

**Pull request merged**